cmd/astro-wasm/astro-wasm.go

```diff
@@ -22,20 +22,22 @@ import (
 var done chan bool
 
 func main() {
-	js.Global().Set("__astro_transform", Transform())
-	// This ensures that the WASM doesn't exit early
-	<-make(chan bool)
+	js.Global().Set("@astrojs/compiler", make(map[string]interface{}))
+	module := js.Global().Get("@astrojs/compiler")
+	module.Set("transform", Transform())
```

Comment:
This is much cleaner. Instead of doing `__astro_transform`, we create `globalThis["@astrojs/compiler"]["transform"]` (and others in the future)